R/adjust.time.R

```diff
@@ -25,15 +105,25 @@ adjust.time <- function(features,
         if(is.na(mz.tol)) {
             mz.tol <- find.tol(mz, uppermost = find.tol.max.d, do.plot = do.plot)
         } else if(do.plot) {
-            plot(c(-1,1), c(-1,1), type="n", xlab="",
-                 ylab="", main="m/z tolerance level given", axes=FALSE)
-            text(x=0, y=0, mz.tol, cex=1.2)
+            draw_plot(x = c(-1,1), 
+                      y = c(-1,1), 
+                      xlab = "", 
+                      ylab = "", 
+                      main = "m/z tolerance level given", 
+                      axes = FALSE, 
+                      label = mz.tol, 
+                      cex = 1.2)
         }
         
         if(!is.na(chr.tol) && do.plot) {
-            plot(c(-1,1), c(-1,1), type="n", xlab="",
-                 ylab="", main="retention time \n tolerance level given", axes=FALSE)
-            text(x=0, y=0, chr.tol, cex=1.2)
+            draw_plot(x = c(-1,1), 
+                      y = c(-1,1), 
+                      xlab = "", 
+                      ylab = "", 
+                      main = "retention time \n tolerance level given", 
+                      axes = FALSE, 
+                      label = chr.tol, 
+                      cex = 1.2)
```

Comment:
And one more time.

Reply:
resolved in [024c3be](https://github.com/RECETOX/recetox-aplcms/pull/64/commits/024c3be853f22d9a0ff5a7a5769194cf9e74eb24)